mac/FreeChat/Models/NPC/LlamaServer.swift

```diff
@@ -269,30 +262,25 @@ actor LlamaServer {
     interrupted = false
     serverErrorMessage = ""
 
-    outputPipe.fileHandleForReading.readabilityHandler = { handle in
-      let data = handle.availableData
-      let lines = String(decoding: data, as: UTF8.self)
-
-      #if DEBUG
-        print(lines)
-      #endif
-
-      Task {
-        await self.handleServerOutput(lines)
-      }
-    }
-
     guard let modelPath = self.modelPath else { return }
     let modelName =
       modelPath.split(separator: "/").last?.map { String($0) }.joined() ?? "Unknown model name"
 
+    let serverHealth = ServerHealth()
+    await serverHealth.updateURL(url("/health"))
+    await serverHealth.check()
+
     var timeout = 60
```

Comment:
Notice that the health check has response time, and HTTP timeout of one second  https://github.com/psugihara/FreeChat/blob/main/mac/FreeChat/Models/NPC/ServerHealth.swift#L17